dataflow-migrate-schedules/README.adoc

```diff
@@ -58,8 +58,6 @@ applications:
     spring_cloud_deployer_cloudfoundry_services: <your SCDF database service,your scheduler service>
     spring_cloud_scheduler_cloudfoundry_schedulerUrl: <URL to scheduler service>
     spring_profiles_active: cf
```

Comment:
I wonder `cf` is used already from the existing migration sample. But, this should also include `cloudfoundry` as the MigrationService is activated based on the profile name `cloudfoundry`.

Reply:
good catch.  I'll update.